ibm/resource_ibm_kms_key_rings_test.go

```diff
@@ -0,0 +1,120 @@
+package ibm
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMKMSResource_Key_Ring_Name(t *testing.T) {
+	instanceName := fmt.Sprintf("kms_%d", acctest.RandIntRange(10, 100))
```

Comment:
instead of kms_%d make it tf_kms_%d
tf_ is the common prefix which we genereally use..
This change applies in all places